compiler/semstmts.nim

```diff
@@ -912,6 +912,8 @@ proc typeSectionLeftSidePass(c: PContext, n: PNode) =
       s.typ.sym = s             # process pragmas:
       if name.kind == nkPragmaExpr:
         pragma(c, s, name.sons[1], typePragmas)
+        if sfExportc in s.flags and a.sons[2].kind == nkIdent:
```

Comment:
And if I write ``Y* {.exportc.} = module.X`` the compiler accepts it. I think we produe ``tyAlias`` for type aliases, could check for that instead. Somehow.

Reply:
Done.